lib/hammer/mnesia.ex

```diff
@@ -68,6 +71,14 @@ defmodule Hammer.Mnesia do
   end
 
   @doc false
+  @spec hit(
+          table :: atom(),
+          key :: term(),
+          scale :: non_neg_integer(),
+          limit :: non_neg_integer(),
+          increment :: non_neg_integer()
+        ) ::
+          {:allow, non_neg_integer()} | {:deny, non_neg_integer()}
```

Comment:
This can get out of sync with the behavior. Also, they don't provide any useful information.